modules/installation-guide/partials/proc_using-minikube-to-set-up-kubernetes.adoc

```diff
@@ -17,5 +17,7 @@ This section describes how to use Minikube to set up a local single-node {kubern
 . Start Minikube (it is important to *allocate at least 4 GB of RAM*):
 +
 ----
-$ minikube start --memory=4096
+$ minikube start --addons=ingress --memory=4096
 ----
++
+NOTE: For Windows and Mac users it is required to add the `--vm=true` option to run Minikube in a virtual machine.
```

Comment:
FYI notes are described with

```
[NOTE]
====
text here
```

Reply:
did you run vale plug-in on the changes ?
you should see error on passive `is required`

Reply:
I've seen the `NOTE:` definition as well in many places. Is there any rule for this?

I didn't run `vale` on my branch (as checklist says) because I have problems with workspace from default devfile.